reactor-core/src/main/java/reactor/core/publisher/LambdaMonoSubscriber.java

```diff
@@ -171,8 +171,7 @@ public final void onNext(T x) {
 				consumer.accept(x);
 			}
 			catch (Throwable t) {
-				Operators.onErrorDropped(t, this.initialContext);
-				return;
+				doError(t);
```

Comment:
this looks pretty much aligned with what is done in `LambdaSubscriber#onNext`, taking into account the fact that for `Mono` it already cancels the subscription (and calls `completeConsumer` on top of that). that said, maybe the only thing missing vs the flux version is `Exceptions.throwIfFatal(t)`